adapters/redis/test/opentelemetry/adapters/redis/adapter_test.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+# Copyright 2020 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'test_helper'
+
+require_relative '../../../../lib/opentelemetry/adapters/redis'
+require_relative '../../../../lib/opentelemetry/adapters/redis/patches/client'
+
+describe OpenTelemetry::Adapters::Redis::Adapter do
```

Comment:
Our tests are looking a little slim compared to those in dd-trace: https://github.com/DataDog/dd-trace-rb/blob/master/spec/ddtrace/contrib/redis/redis_spec.rb. Make sure we have an equivalent test for everything that applies. I realize we might not have ported everything, but there is a test for spans generated from the auth command, for example, that I think we should preserve.